src/common/hooks/useInvoiceMutation/useInvoiceMutation.test.js

```diff
@@ -30,9 +30,11 @@ describe('useInvoiceMutation', () => {
     );
 
     await act(async () => {
-      result.current.mutateInvoice({
-        status: 'Open',
-      });
+      result.current.mutateInvoice(
+        {
+          invoice: { status: 'Open' },
```

Comment:
Please don't use a "magic" strings/numbers, there is a `INVOICE_STATUS` dictionary for invoice statuses